### ISSUE TITLE
Помогите с генераторм

### ISSUE THREAD

**Issue by koalex** [bem-site/bem-forum-content-ru#42]
Всем привет! Когда запускаю генератор, то на вопросе "Use design from bem-components? YES" вылетает такая ошибка http://floomby.ru/s2/hW686z
Что это значит? Вроде пишет что нет какого-то модуля, вообщем не могу разобраться..

**Comment by weynemeynen** [comment]
после настройки проекта с помощью generator-bem-stub, при выборе: enb, bh, не работает блок "selector" из bem-components
в project-stub с bemhtml работает.

причина: не собирается файл .js

**Comment by eGavr** [comment]
Какая версия генератора используется?

**Comment by weynemeynen** [comment]
generator-bem-stub@0.12.0

**Comment by eGavr** [comment]
Могу предположить, что ты выбрал немного не те ответы на вопросы. Если тебе нужнен `enb + bem-components + bh + js`, то ответы должны быть такими (проверял):

![2015-11-09 10 26 53](https://cloud.githubusercontent.com/assets/6376693/11028088/79a53cd8-86cc-11e5-80cb-64ae8896cfba.png)

Важно при выборе технологий указать `browser.js`.

**Comment by weynemeynen** [comment]
заработало!
ответы выбирал те же, кроме browser.js
И ещё момент, не пресуются css, js, bh.js (выбрано ...to be minimized)

А так хорошая штука generator-bem-stub, экономит кучу времени.
Спасибо!

**Comment by eGavr** [comment]
Чтобы файлы минимизировались, надо указать переменную среды окружения `YENV = production`, то есть как-то так:

```bash
YENV=production ./node_modules/.bin/enb make
```

**Comment by weynemeynen** [comment]
ОК! Я полагал, значение 'production ' можно прописать в make.js чтобы сжимало по умолчанию

**Comment by Guria** [comment]
@weynemeynen вы можете добавить запись в секцию `scripts` вашего `package.json` файла для запуска сборки в `production` режиме:
```js
"scripts": {
  "build": "YENV=production enb make"
}
```
* https://docs.npmjs.com/files/package.json#scripts
* https://docs.npmjs.com/misc/scripts

**Comment by eGavr** [comment]
> ОК! Я полагал, значение 'production ' можно прописать в make.js чтобы сжимало по умолчанию

Да, так тоже можно сделать, чтобы сжимало по умолчанию. Посмотри в файлик `make.js`, там обычный js и ничего больше, там можно увидеть, что нужно поменять, чтобы всегда сжималось.

**Comment by weynemeynen** [comment]
Да, спасибо, package.json ОК!
